sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -3,27 +3,89 @@
 
 #pragma once
 
+#include <string>
+#include <vector>
+
 #include <internal/contract.hpp>
 
+using namespace std;
```

Comment:
never use namespaces in headers

Reply:
For the uninitiated, can you please explain why?

Reply:
Because you include azure header file and all of a sudden in your code everything from the std namespace becomes available without the namespace prefix.
`using namespace x`, if ever, should only be used in .cpp files (except for the rare exceptions where you explicitly want it to do what it is doing, which is unlikely ever a case).

Reply:
Fixed it.
Yeah, I added it there so I didn't need to use it again on every .cpp file, but yeah, not everyone would like that.
thank you